### PR TITLE
Use codesandbox demos instead of netlify

### PR DESCRIPTION
**Pull request by yakhinvadim**
This PR changes all our links to stable demos from Netlify to Codesandbox pointing at our GitHub repo.
Also updated the Overview page in docs to embed a live demo:

https://user-images.githubusercontent.com/11838280/166825932-2083380b-8f67-4af7-85f1-a602bf467d28.mp4

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _docs-elastic-co_ successfully!

* https://docs-elastic-hyco4wnyc-elastic-dev.vercel.app

* Built with commit 523a2711f8f4b8fab4742765ff6a0859a40984a9

> Issues? Visit #next-docs in Slack

File: docs/overview.mdx [hunk 1]
```diff
@@ -24,9 +24,17 @@ Use it with [Elastic](https://www.elastic.co/) to have a search experience up an
 
 ### Live Demo
 
-Check out the [live demo of Search UI](https://search-ui-stable.netlify.com).
-
-[![Edit search-ui-national-parks-example](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/search-ui-national-parks-example-ts-k6u5iz)
+<iframe
+  src="https://codesandbox.io/embed/github/elastic/search-ui/tree/master/examples/sandbox?autoresize=1&fontsize=12&initialpath=%2Felasticsearch&module=%2Fsrc%2Fpages%2Felasticsearch%2Findex.js&theme=light&view=preview&hidedevtools=1"
```
Comment (JasonStoltz) [inline docs/overview.mdx:28]: Should this be from "stable" or "master"?

Comment (yakhinvadim) [reply docs/overview.mdx:28]: Nice catch, should be stable.

Comment (yakhinvadim) [inline docs/overview.mdx:28]:
```suggestion
  src="https://codesandbox.io/embed/github/elastic/search-ui/tree/stable/examples/sandbox?autoresize=1&fontsize=12&initialpath=%2Felasticsearch&module=%2Fsrc%2Fpages%2Felasticsearch%2Findex.js&theme=light&view=preview&hidedevtools=1"
```

**Comment by yakhinvadim** [comment]
> I like the idea, but it feels a bit squished.

I also think it's a bit squished, but I think it still provides a lot of value.

We could ask the docs team to implement a "FullWidth" container that would give us all the available space for inline demos.

**Comment by goodroot** [comment]
> We could ask the docs team to implement a "FullWidth" container that would give us all the available space for inline demos.

Very open to feedback... & PRs!

**Pull request merged**